cdntaxreceipts.functions.inc

```diff
@@ -618,7 +610,7 @@ function cdntaxreceipts_configure_inkind_fields() {
     $group = array(
       'title' => 'In-kind donation fields',
```

Comment:
What fields are supposed to go into this group, and why is that not part of this PR? On test2, when I create a contribution with fund of In-kind, then this field set opens and shows

DESCRIPTION OF PROPERTY * |  
APPRAISED BY * |  
ADDRESS OF APPRAISER * |  
ORIGINAL COST